web/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import LoginScreen from "./components/LoginScreen";
+import TaskApp from "./TaskApp";
+import RealmApolloProvider from "./graphql/RealmApolloProvider";
+import { useRealmApp, RealmAppProvider } from "./RealmApp";
+
+export const APP_ID = "tasktracker-huhcb";
+
+const LoggedIn = ({ children }) => {
+  // Only render children if there is a logged in user.
+  const {app} = useRealmApp();
+  return app.currentUser ? children : null;
+};
+
+const LoggedOut = ({ children }) => {
+  // Only render children if there is not a logged in user.
+  const {app} = useRealmApp();
+  return app.currentUser ? null : children;
+};
+
+export default function App() {
```

Comment:
some of the app uses function syntax and the rest of the app uses es6 arrow function syntax. I think it might be a good idea to stick to one convention unless you have a particular reason for not doing so.